docker-compose.yml

```diff
@@ -66,6 +66,35 @@ services:
     depends_on:
       database:
         condition: service_healthy
+
+  ##########################################################
+  #                      FrontEnd                          #
+  ##########################################################
+  frontend:
+    container_name: quackStack-frontend
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile.dev
+    develop:
+      watch:
+        - action: sync
+          path: ./frontend
+          target: /frontend
+          ignore:
```

Comment:
Use a .dockerignore file instead of this.  It'll be used in builds and cleaned up in OpenShift as well.  node_modules is good there though!